testsuite/MDAnalysisTests/analysis/test_hole.py

```diff
@@ -21,27 +21,22 @@
 #
 from __future__ import print_function, absolute_import
 
-from six.moves import range
+import errno
 
 import MDAnalysis
 import MDAnalysis.analysis.hole
+import matplotlib
+import mpl_toolkits.mplot3d
+import numpy as np
+import pytest
 from MDAnalysis.analysis.hole import HOLEtraj, HOLE
-
-from numpy.testing import (TestCase, dec,
-                           assert_equal, assert_almost_equal,
+from MDAnalysisTests import (executable_not_found, tempdir, in_dir)
+from MDAnalysisTests.datafiles import PDB_HOLE, MULTIPDB_HOLE
+from numpy.testing import (TestCase, assert_equal, assert_almost_equal,
                            assert_array_equal,
                            assert_array_almost_equal, assert_)
-import numpy as np
-import matplotlib
-import mpl_toolkits.mplot3d
-
-
-import os
-import errno
+from six.moves import range
 
-from MDAnalysisTests.datafiles import PDB_HOLE, MULTIPDB_HOLE
-from MDAnalysisTests import (executable_not_found, module_not_found,
-                             tempdir, in_dir)
```

Comment:
import order is  

1. python std library
1. external packages like numpy/pandas/...
1. mdanalysis specific packages
1. testing specific packages